priv/repo/migrations/20201118162938_add_timestamps_to_chat.exs

```diff
@@ -0,0 +1,9 @@
+defmodule Glimesh.Repo.Migrations.AddTimestampsToChat do
+  use Ecto.Migration
+
+  def change do
+    alter table(:users) do
+      add :show_timestamps?, :boolean, default: false
```

Comment:
Does this result in having the column actually called `show_timestamps?`? If so I think we should change it, as it would need to be escaped when using it a select statement. It's also very non-traditional :P

Reply:
Yeah it does, I'll swap it over to not include the ?